src/libraries/Microsoft.PowerFx.Core/Microsoft.PowerFx.Core.csproj

```diff
@@ -30,11 +31,11 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<EmbeddedResource Include="..\..\strings\**\PowerFxResources.resw" Condition="'$(Configuration)' == 'Debug'">
-			<Link>Strings\%(RecursiveDir)%(Filename)%(Extension)</Link>
+		<EmbeddedResource Include="..\..\strings\PowerFxResources.*.resx" Condition="'$(Configuration)' == 'Debug'">
+			<Link>strings\%(Filename)%(Extension)</Link>
 		</EmbeddedResource>
-		<EmbeddedResource Include="..\..\strings\**\PowerFxResources.resw" Exclude="$(SolutionDir)\strings\qps-ploc\**" Condition="'$(Configuration)' == 'Release'">
-			<Link>Strings\%(RecursiveDir)%(Filename)%(Extension)</Link>
+		<EmbeddedResource Include="..\..\strings\PowerFxResources.*.resx" Exclude="..\..\strings\PowerFxResources.qps-ploc.resx" Condition="'$(Configuration)' == 'Release'">
```

Comment:
Resx files with this naming pattern get converted to satellite assemblies by default